robottelo.properties.sample

```diff
@@ -79,3 +79,44 @@ driver=firefox
 # Default set to be 0, i.e. no timing of performance is measured and thus no
 # interference to original robottelo tests.
 test.foreman.perf=0
+
+# Folowing entries are used for preparation of performance tests after a fresh
+# install. They will be used by
+# `test/foreman/performance/test_standard_prep.py`, which supports:
+#
+# 1. downloading manifest,
+# 2. uploading manifest to subscription,
+# 3. updating Red Hat CDN URL,
+# 4. enabling key repositories: rhel6-rpms, rhel7-rpms, rhel6-kickstart-rpms,
+#    rhel7-kickstart-rpms, rhel6-optional-rpms, rhel7-optional-rpms,
+#    rhel6-optional-source-rpms, rhel7-optional-source-rpms,
+#    rhel6-optional-debug-rpms, r7-optional-debug-rpms
+#
+# Note that this preparation step is not required as long as satellite server
+# is already configured.
+test.cdn.address=http://example.redhat.cdn/pub
```

Comment:
If an argument is optional, please comment it out. If it's required, then simply leave it blank. Doing so allows code to fail more quickly and more severely:

* An exception will be thrown if this option is unset and code reads it.
* Code which reads this option and can deal with unset values can explicitly test for a value of `''`.